install-deps.sh

```diff
@@ -13,6 +13,7 @@
 #  version 2.1 of the License, or (at your option) any later version.
 #
 set -e
+docker run --rm -it --entrypoint promtool prom/prometheus:v2.9.2
```

Comment:
This should presumably be `docker pull prom/prometheus:v2.9.2`. `docker run --rm -it ...` does also pull the image but also runs the container with the `promtool` executable and no arguments to `promtool`. The purpose would be clearer if it'd just pull, IMHO.

Reply:
Will do that. Thanks